packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -854,9 +858,22 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         private _storage?: IDocumentStorageService,
     ) {
         super();
-
         this.baseSummaryMessage = metadata?.message;
 
+        /**
```

Comment:
nit - I'm not used to seeing this style block comment except for TSDoc comments for functions, classes, etc.  I think just line comments // are good for this kind of thing.